### PR TITLE
added HTML q5

### PR DESCRIPTION
**Pull request by Dhanu084**

### CODE REVIEW

File: html/answers/dhanush-q4.html [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+                    <h4>Explain the importance of the HTML DOCTYPE?</h4>
+                </h4>
+            </div>
+            <br>
+            <br>
+            <article>
+                <strong>Answer:</strong> 
+                <p>
+                    <ul>
+                        <li>DOCTYPE is an instruction to the web browser regarding what version of the markup language the page is written. </li>
```
Comment (shivamyth) [inline html/answers/dhanush-q4.html]: We got an almost same question earlier, add a different one.

**Review by shivamyth — CHANGES_REQUESTED** [review]

File: html/answers/dhanush-q4.html (hunk 1 above)
Comment (Dhanu084) [reply html/answers/dhanush-q4.html]: Changes done

Comment (Dhanu084) [reply html/answers/dhanush-q4.html]: Resolved Q4 conflict as I changed my questio no to 5

**Comment by dakshkhetan** [comment]
@Dhanu084 Hey, please take a fresh pull (from upstream development branch) and then add your question.
Since this PR is showing 9 files changed.

**Comment by dakshkhetan** [comment]
Closing this PR. You can raise another PR after making the requested changes.

**Pull request closed without merging**